site/src/ThemeSelector.jsx

```diff
@@ -119,13 +132,13 @@ const ThemeSelector = ({
           }
         }}
         sx={{
-          marginTop: "-7px",
+          marginTop: "-2px",
```

Comment:
This change centers the pin icons a bit with their respective rows- there's probably a less kludgy way to do this adjustment.